shared/customui/customui.cpp

```diff
@@ -39,132 +38,114 @@ namespace CustomUI
         Array<Il2CppObject *> *allObjects;
 
         // Find Objects of type TMP_fontAsset
-        if (!RunMethod(&allObjects, nullptr, FindMethod("UnityEngine", "Resources", "FindObjectsOfTypeAll", 1), type_fontasset))
-        {
+        if (!RunMethod(&allObjects, "UnityEngine", "Resources", "FindObjectsOfTypeAll", type_fontasset)) {
             // EXCEPTION
             log(ERROR, "TextObject::create: Failed to Find Objects of type TMP_fontAsset");
             return false;
         }
         int match = -1;
-        for (int i = 0; i < allObjects->Length(); i++)
-        {
+        for (int i = 0; i < allObjects->Length(); i++) {
             // Treat it as a UnityEngine.Object (which it is) and call the name getter
             Il2CppString *assetName;
-            if (!RunMethod(&assetName, allObjects->values[i], "get_name"))
-            {
+            if (!RunMethod(&assetName, allObjects->values[i], "get_name")) {
                 // EXCEPTION
                 log(ERROR, "TextObject::create: Failed to run get_name of assetName");
                 return false;
             }
-            if (strcmp(to_utf8(csstrtostr(assetName)).c_str(), "Teko-Medium SDF No Glow") == 0)
-            {
+            if (strcmp(to_utf8(csstrtostr(assetName)).c_str(), "Teko-Medium SDF No Glow") == 0) {
```

Comment:
Should transition away from these methods and instead use the std::string equals operator (given that this is a string_view, it should behave as such)

Reply:
Ah, good catch